projects/Mallard/src/screens/settings-screen.tsx

```diff
@@ -127,6 +130,10 @@ const DevZone = withNavigation(({ navigation }: NavigationInjectedProps) => {
     )
 })
 
+const rightChevronIcon = (
+    <RightChevron textColor={useAppAppearance().color} fontSize={22} />
```

Comment:
I wouldn't outright pass the font size here because it's too much of an implementation detail. if you really wanna expose a size modifier I'd do an optional multiplier parameter?

Reply:
Have removed props and kept the icon defaults inside the icon component for now.